### ISSUE TITLE
How is this different than GPT-2

### ISSUE THREAD

**Issue by bladedsupernova** [facebookresearch/ELI5#16]
What's different?

**Comment by huihuifan** [comment]
Hi, GPT-2 is a pretrained language model. This is a dataset for long form
question answering. Our task challenges models to answer open ended and
complex questions that require explanatory, paragraph length answers by
leveraging web queries from common crawl.

**Comment by bladedsupernova** [comment]
But GPT-2 seems to do just that...

Do you mean you are cutting/pasting snippets together to create a new abstract paragraph?

Does yours generate word by word? Or sentence by sentence?

**Comment by huihuifan** [comment]
GPT-2 is a model, ELI5 is a dataset.
